src/sparsezoo/utils/node_inference.py

```diff
@@ -81,7 +81,9 @@ def extract_nodes_shapes_and_dtypes_ort(
 
     sess_options = onnxruntime.SessionOptions()
     sess_options.log_severity_level = 3
-    sess = onnxruntime.InferenceSession(model_copy.SerializeToString(), sess_options)
+    sess = onnxruntime.InferenceSession(
+        model_copy.SerializeToString(), sess_options, providers=["CPUExecutionProvider"]
```

Comment:
Approving but noting it would be nice to have another helper in sparsezoo similar to get_ort_session_from_model, this line would break with Llms>2GB